tests/st2tests.yaml

```diff
@@ -21,7 +21,7 @@ services:
     depends_on:
       - st2test-tools
   st2test-tools:
-    image: dduportal/bats:latest
+    image: zoeleah/bats:latest
```

Comment:
Let's use the official `bats/bats` image, which has a community behind it

Reply:
remove the hole bats container and do an installation for bats with `apt install bats`.